sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatThreadAsyncClient.java

```diff
@@ -271,14 +272,14 @@ public void listChatParticipantsPages(String chatThreadId, Integer maxPageSize,
      * Remove a participant from a thread.
      *
      * @param chatThreadId Thread id to remove the participant from.
-     * @param chatParticipantId Id of the thread participant to remove from the thread.
+     * @param participantCommunicationIdentifier Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set.
      * @param callback the Callback that receives the response.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws CommunicationErrorResponseException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      */
-    public void removeChatParticipant(String chatThreadId, String chatParticipantId, final Callback<Void> callback) {
-        this.serviceClient.removeChatParticipant(chatThreadId, chatParticipantId, callback);
+    public void removeChatParticipant(String chatThreadId, CommunicationIdentifierModel participantCommunicationIdentifier, final Callback<Void> callback) {
```

Comment:
Are you planning to do the serialization/deserialization of `CommunicationIdentifierModel` ↔ `CommunicationIdentifier` in a separate PR?

Reply:
none of the identitifet"Model" types should be exposed by the end.

Reply:
Why do we need to do this conversion do you know?

Reply:
I see, so user never uses CommunicationIdentifierModel? then I will need to create a converter class for that.

Reply:
There is a already a converter class, `CommunicationIdentifierSerializer`, you do not make a new one.

Reply:
https://github.com/Azure/azure-sdk-for-ios/pull/733/files#diff-306cf97a4c370a90504a12c5ac43ae67d8e9f335ae065307fda64402179a23efR429 This is an example of what was done in the iOS repo you can use as an example

Reply:
I checked code, it involves quite a lot of changes for callbacks and sync calls. From preview5, azure core team will release changes that duplicate ChatParticipant, ChatMessage, etc models to auto generated and handwritten, which support the conversion between CommunicationIdentifier and CommunicationIdentifierModel easier. 

For preview 4, can we go it as this or do I have make this change?

Reply:
Let's do it in a separate PR then